src/expr.rs

```diff
@@ -194,6 +194,40 @@ impl<'a> FmtVisitor<'a> {
         format!("{}: {}", name, expr)
     }
 
+    fn rewrite_tuple_lit(&mut self, items: &[ptr::P<ast::Expr>], width: usize, offset: usize)
+        -> String {
+        // opening paren
+        let indent = offset + 1;
+        // Only last line has width-1 as budget, other may take max_width
+        let item_strs: Vec<_> =
+            items.iter()
+                 .enumerate()
+                 .map(|(i, item)| self.rewrite_expr(
+                    item,
+                    // for last line, -2 is for indent + ")", for other lines, -1 is for comma
```

Comment:
I think the 'other lines' should also be -2 because they are also offset by the opening `(`